Packs/NistNVD/Integrations/NistNVD/NistNVD.py

```diff
@@ -0,0 +1,322 @@
+    startIndex = demisto.args().get('startIndex')
+    resultsPerPage = demisto.args().get('resultsPerPage')
+    additional_parameters = '?modStartDate=' + start_date + ' UTC-05:00' + \
+        '&startIndex=' + str(startIndex) + '&resultsPerPage=' + str(resultsPerPage)
+    generalSearchRequest = connection(base_url, additional_parameters)
+    demisto.results(generalSearchRequest)
+    generalVulnerabilityList = extractVulnDetails(generalSearchRequest)
+
+    headers = ['CVE ID', 'Description', 'Published Date', 'Last Modified Date', 'References']
+    hr = tableToMarkdown('Vulnerabilities', generalVulnerabilityList, headers, removeNull=True)
+
+    return {
+        'Type': entryTypes['note'],
+        'Contents': generalVulnerabilityList,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': hr
+    }
+
+
+def keywordSearch():
+
+    base_url = urljoin(demisto.params()['url'], '/rest/json/cves/1.0')
+    keyword = demisto.args().get('keyword')
+    isExactMatch = demisto.args().get('isExactMatch')
+    time = int(demisto.args().get('time'))
+    last_time = datetime.today() - timedelta(hours=int(time))
+    start_date = last_time.strftime('%Y-%m-%dT%H:%M:%S:000')
+    startIndex = demisto.args().get('startIndex')
+    resultsPerPage = demisto.args().get('resultsPerPage')
+    additional_parameters = '?modStartDate=' + start_date + ' UTC-05:00' + '&keyword=' + keyword + \
+        '&isExactMatch=' + isExactMatch + '&startIndex=' + str(startIndex) + '&resultsPerPage=' + str(resultsPerPage)
+    # additional_parameters = '?keyword=' + keyword + '&isExactMatch=' + isExactMatch
+    generalSearchRequest = connection(base_url, additional_parameters)
+    # demisto.results(generalSearchRequest)
+    generalVulnerabilityList = extractVulnDetails(generalSearchRequest)
+
+    headers = ['CVE ID', 'Description', 'Published Date', 'Last Modified Date', 'References']
+    hr = tableToMarkdown('Vulnerabilities', generalVulnerabilityList, headers, removeNull=True)
+
+    return {
+        'Type': entryTypes['note'],
+        'Contents': generalVulnerabilityList,
+        'ContentsFormat': formats['json'],
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': hr
+    }
+
+
+def cvssSearch():
+
+    base_url = urljoin(demisto.params()['url'], '/rest/json/cves/1.0')
+    time = int(demisto.args().get('time'))
+    last_time = datetime.today() - timedelta(hours=int(time))
+    start_date = last_time.strftime('%Y-%m-%dT%H:%M:%S:000')
+
+    cvssType = demisto.args().get('cvssType')
+    key = demisto.args().get('key')
+    searchParameters = cvssType + key
+    value = demisto.args().get('value')
+
+    startIndex = demisto.args().get('startIndex')
+    resultsPerPage = demisto.args().get('resultsPerPage')
+
+    additional_parameters = '?modStartDate=' + start_date + ' UTC-05:00' + '&' + searchParameters + \
```

Comment:
Why is it `UTC-05:00` for the start date?

Reply:
I used UTC according to documentation but I changed from UTC-05 to UTC-00

Reply:
It would also be a good thing to add in the documentation and the `additionalInfo` of the argument that it's UTC-00 time.